source/extensions/filters/http/jwt_authn/verifier.cc

```diff
@@ -16,19 +16,19 @@ class BaseVerifierImpl : public Verifier {
 public:
   BaseVerifierImpl(const BaseVerifierImpl* parent) : parent_(parent) {}
 
-  void onCompleteHelper(Status status, VerifyContext& context) const {
+  void onCompleteHelper(Status status, VerifyContextSharedPtr& context) const {
```

Comment:
In general,  don't pass & for share_ptr.   just use "const VerifyContext&" directly,  if there is lambda function inside that need to increase the reference_count,  then pass in share_ptr.   but not its &